src/util/clickTrackingReader.js

```diff
@@ -1,11 +1,11 @@
-import { parseMemberCookie } from './cookieUtils';
+import config from './config';
 
-const isProd = process.env.NODE_ENV === 'production';
+import { parseMemberCookie } from './cookieUtils';
 
 export const clickCookieOptions = {
```

Comment:
let's make this a function that can be passed a `server` instance, so that the `isProd` can be read from `server.app` rather than an import. `clickCookieOptions` is only called from places where we have access to `request.server`, so it should be fine.